modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -33,9 +33,9 @@ def submit
           end
 
           ews = create_ews(params[:id])
-          ClaimsApi::EvidenceWaiverBuilderJob.perform_async(ews.id)
+          ClaimsApi::EvidenceWaiverBuilderJob.perform_async(ews.id, bgs_claim)
 
```

Comment:
It looks like `bgs_claim` is not used in the sidekiq job anywhere.  If that is the case I would say it is probably best not to change the number of arguments the job takes in.